ocaml/xapi/xapi_vm.ml

```diff
@@ -234,6 +244,9 @@ let start ~__context ~vm ~start_paused ~force =
     debug "Setting ha_always_run on vm=%s as true during VM.start" (Ref.string_of vm)
   end;
 
+  if not force && not start_paused then
```

Comment:
Because a VM on nested virt cannot suspend/resume. But because this cannot happen, maybe I should not test for this and rely on the fact that suspend/resume is impossible.